torch/_decomp/__init__.py

```diff
@@ -18,6 +18,10 @@
 
 meta_lib = torch.library.Library("aten", "IMPL", "Meta")
 
+# decompositions which have been disabled as meta kernel implementations,
+# usually due to mismatching strides, aliasing, or other inconsistent property
```

Comment:
@SherlockNoMad @Chillee this is something we're going to have to resolve before we can move to the single global table... we can't have wrong implementations